### PR TITLE
move domain ES pillow to kafka

### PR DESCRIPTION
**Pull request by czue**
:fish: or :office: 

don't merge yet - going to put this on staging and make sure everything is working.

@snopoke cc @nickpell

### CODE REVIEW

File: corehq/apps/hqcase/management/commands/ptop_preindex.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
-from gevent import monkey; monkey.patch_all()
-from corehq.elastic import get_es_new
+from gevent import monkey;
```
Comment (dimagimon) [inline corehq/apps/hqcase/management/commands/ptop_preindex.py]: E703 statement ends with a semicolon

File: corehq/apps/hqcase/management/commands/ptop_preindex.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+from corehq.pillows.utils import get_all_expected_es_indices
 
-from pillowtop.es_utils import get_all_inferred_es_indices_from_pillows
+monkey.patch_all()
```
Comment (snopoke) [inline corehq/apps/hqcase/management/commands/ptop_preindex.py]: Isn't the monkey patching handled in `manage.py`?

Comment (czue) [inline corehq/apps/hqcase/management/commands/ptop_preindex.py]: hmm, maybe. I think that's just been in that file for ages. will investigate

**Comment by snopoke** [comment]
:+1: once QA passes

**Comment by nickpell** [comment]
have to ask - where does :office: come from?

**Comment by czue** [comment]
@nickpell I think it's just like a fully constructed building. so all put together (all at once).

**Comment by nickpell** [comment]
aha :bulb:

**Comment by czue** [comment]
QA is looking good but let's not merge until tomorrow just in case.

**Comment by czue** [comment]
@snopoke I'm keen to get this merged (I can deploy now and monitor prod just in case any issues). Also will be nice to get https://github.com/dimagi/commcare-hq/pull/10981 in which now depends on this.

**Comment by czue** [comment]
(added one commit since you reviewed which you should check out, but otherwise :white_check_mark: from my side)

**Pull request merged**